discord/ui/dynamic.py

```diff
@@ -165,6 +170,14 @@ def row(self) -> Optional[int]:
     def row(self, value: Optional[int]) -> None:
         self.item.row = value
 
+    @property
+    def disabled(self) -> bool:
```

Comment:
```suggestion
    def disabled(self) -> Optional[bool]:
```

Reply:
This one is not `Optional` since it comes from the `item` itself. There `False` is default.